assets/css/style.css

```diff
@@ -18,9 +18,11 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
```

Comment:
If you want to change the font please change it in the body, where it is already set, and not here. You can also remove the import as this will no longer be necessary.